pipe/section/section_impls/mycelial_server/src/destination.rs

```diff
@@ -173,7 +180,10 @@ impl Mycelial {
     }
 
     fn basic_auth(&self) -> String {
```

Comment:
Eventually we should think about pulling all of the auth stuff together, this looks like the same method from http_client.rs.